### ISSUE TITLE
Icon request

### ISSUE THREAD

**Issue by datamadsen** [SoichiroYamane/sketchybar-app-font-bg#11]
Hi - I am not super proficient in creating svgs for this kind of thing, so I thought I would try and ask here. Please just close the "issue" if it's not something you want to do.

What I would like is the icons for claude and pixelmator if possible.

**Comment by SoichiroYamane** [comment]
I am happy to hear your request! I will draw claude and pixelmator icons!

**Comment by datamadsen** [comment]
Thanks so much :)

**Comment by SoichiroYamane** [comment]
I added claude and pixelmator icons! [751b3cf](https://github.com/SoichiroYamane/sketchybar-app-font-bg/pull/12/commits/751b3cf37d0b15c71f2631e558d6dab66f4bc3ba)

Can you check following lists?

1. Designs of icons are okay for you?
2. If you are using my dotfiles, you can use @SoichiroYamane/dotfiles/sketchybar/icon_updater.sh
3. After you update icons, icons are properly rendered? Also, you can check if the visibility of created icons is enough for your sketchybar.
 
If you have any trouble, please feel free to ask me!

**Comment by datamadsen** [comment]
I think it's great! I discovered that Pixelmator is actually called Pixelmator Pro - so I fixed that in the `mappings/:pixelmator:` in a local fork and then the pixelmator icon works. I added an alias in https://github.com/SoichiroYamane/sketchybar-app-font-bg/pull/13 - actually I am not sure if there is a pixelmator non-pro (not being sold as far as I can see), so maybe you want to handle it a little differently.
